src/client/src/components/JobListing.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import axios from "axios"
+import "../css/jobListing.css";
+import {useEffect, useState} from "react";
+import {Link} from 'react-router-dom'
+
+
+const JobListing = () =>{
+    const [jobs, setJobs] = useState([]);
+
+    useEffect( () => {
+        fetchData()
+    }, [])
+
+    const fetchData = async () => {
+        const {data} = await axios.get("http://localhost:9000/jobs")
+
+        setJobs(data)
+    }
+
+    
+    return(
+            
+        <div className="Container">
+            
+            
+            <div data-testid = "jobPostsContainer" className="jobPosts">
+                <div className="heading">
+                    <b>Job Posts</b>
+                </div>
+
+                {jobs.map(job => (
+                
+                    <div key = {job._id} className="jobPost">
+                        
+                        <p> Job id: {job.job_id}</p>
+                        <p>Title: {job.title}</p>
+                        <p>Description: {job.description}</p>
+                        <p>Salary: {job.salary}</p>
+                        <p>Company: {job.company}</p>
+                        <p>Category: {job.category}</p>
+                        <Link to = {`String text ${job.job_id}`} state = {{jobState:job}}>
```

Comment:
Is `String text` necessary? It is displayed on the address bar
![image](https://user-images.githubusercontent.com/78051064/219801883-3c1f4193-c1f0-40ff-9d75-0ebb29498a95.png)

Reply:
You're right, it was a typo thanks for catching
![Screen Shot 2023-02-17 at 5 33 42 PM](https://user-images.githubusercontent.com/14958097/219808384-18919b96-b2ee-4e4b-b489-7cbbd088857a.png)